pkg/chartutil/dependencies.go

```diff
@@ -216,9 +216,33 @@ func processImportValues(c *chart.Chart) error {
 		return err
 	}
 	b := make(map[string]interface{})
-	// import values from each dependency if specified in import-values
+	// export values to each dependency if specified in export-values
```

Comment:
Some modification here as follows:
- Move this comment to above : `for _, riv := range r.ExportValues {`
- Add a comment here similar to:  `// handle export and import values per dependency`